public/index.html

```diff
@@ -0,0 +1,45 @@
+ <!DOCTYPE html>
+ <html lang="en">
+ <head>
+     <meta charset="UTF-8">
+     <title>Document</title>
+     <link rel="stylesheet" type="text/css" href="styles/game_of_life.css">
+     
+ </head>
+<body id="body">
+    <form>
+        <label for="fname">Columns</label><br>
+        <input type="text" id="columns" name="wid" value=""><br>
+        <label for="lname">Rows</label><br>
+        <input type="text" id="rows" name="hei" value=""><br><br>
+        <button type="button" id="submit-button">Submit</button>
+        <button type="button" id="simulate-button">Simulate</button>
+        <button type="button" id="clear-button">Clear</button>
+        <label for="animate" class="animateLabel">Animate
+            <input type="checkbox" id="animate" class="checkbox" type="checkbox" value="1" autocomplete="off">
+            <span class="checkbox"></span>
+        </label>
+        <div class="presets">
+            <button type="button" class="presetsBtn" id="drawPreset-button">Draw preset</button>
+            <select class="presets-content" id="select">
+                <option class="preset1" id="preset1" value="0"  selected="selected" >Select preset</option>
+                <option class="preset1" id="preset1" value="1" >Toad</option>
+                <option class="preset1" id="preset1" value="2" >Acorn </option>
+                <option class="preset1" id="preset1" value="3" >LWSS</option>
+               
+            </select>
+        </div>
+    </form>
+
+    <p>
+        <span id = "mouse-x"></span>
+        <span id = "mouse-y"></span>
+    </p>
+
+    <canvas id="canvas" style="background:  #ffffff;"></canvas>
```

Comment:
Inline styles are a bad practice. Move this style to the CSS file that you've created